Polyfills/Canvas/Source/nanovg_babylon.cpp

```diff
@@ -42,12 +42,11 @@ BX_PRAGMA_DIAGNOSTIC_IGNORED_MSVC(4244) // warning C4244: '=' : conversion from
 #include "fs_nanovg_fill.bin.h"
```

Comment:
I would recommend not formatting this file at all since it's a copy of another file from nanovg.

Reply:
I've reverted the file to master version.

Reply:
This got formatted again. Maybe put some comments to avoid formatting.

Reply:
Revered the changes and added // clang-format off to the top of the file.